sidecar2cdxj.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import ast
+import json
+import os
+import re
+
+import surt
+from warcio.archiveiterator import ArchiveIterator
+from warcio.timeutils import iso_date_to_timestamp
+
+
+def create_sidecar_cdxj(sidecar_file, archive_dir):
```

Comment:
I just noticed we should add a docstring to this function.

Reply:
How does this sound...

    """This function loads WARC record data from a metadata sidecar into a CDXJ.
    
    The function will take each WARC record from a metadata sidecar, convert the URI
    and WARC-Date into a SURT-formatted URL and timestamp, collect the fields from the
    payload, and combine it all into one single line and write it into the CDXJ.
    """

Reply:
We could leave off "This function" and start with an action...
```
    """Create a CDXJ index from a WARC formatted metadata sidecar file.
    
    Iterates the metadata records of a WARC metadata sidecar file,
    and writes a line to a CDXJ file containing the SURT-formatted URI,
    timestamp, and JSON data block representing the record's payload
    key value pairs.
    Keyword arguments:
    sidecar_file -- path to sidecar metadata WARC
    archive_dir -- path to output directory for the CDXJ
    """
```